builder/bundles.go

```diff
@@ -609,7 +609,7 @@ lib=%s
 src=%s
 `, cfg.DebugInfoBanned, cfg.DebugInfoLib, cfg.DebugInfoSrc)
 	}
-	err = ioutil.WriteFile(filepath.Join(b.StateDir, "server.ini"), serverINI.Bytes(), 0644)
+	err = ioutil.WriteFile(filepath.Join(b.Config.Builder.ServerStateDir, "server.ini"), serverINI.Bytes(), 0644)
```

Comment:
Future patch, `build update` should just call the original conf instead of relying on this. This isn't a change required for this PR though.

Reply:
Yep, the idea is to get rid of the other conf files. I hope to get this on with my next patch.